demos/Agent_Based_Modeling/prey_predator_abm/model/partial_state_update_block.py

```diff
@@ -66,7 +66,7 @@
         'policies': {
             'natural_death': natural_death
         },
-        'variables': {
+        'states': {
```

Comment:
H. needed for labs - use 'states' instead of 'variables'. will make backwards compatible later